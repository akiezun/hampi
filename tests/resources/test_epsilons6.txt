S = "a" B "z" ;
B = | "b" B ;