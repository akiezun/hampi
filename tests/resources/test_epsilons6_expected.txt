B = "b"  | "b" B  ;
S = "a" "z"  | "a" B "z"  ;